README.md

```diff
@@ -67,7 +94,21 @@ Belle is friends with Mrs. Potts
 ```
 
 ```rb
-# Your code goes here...
+
+require "pry"
+
+class Belle
+
+@friends = ["Chip Potts", "Cogsworth", "Lumière", "Mrs. Potts"]
+
+def say_friends
+  puts "Belle is friends with #{@friends.each}"
+end
+
+say_friends
+end
+
+binding.pry
```

Comment:
Again, don't need to define a class for this. Just need to call `.each` on the array provided in the question. So something like this...

```rb
friends.each do |friend|
  puts "Belle is friends with #{friend}"
end
```